src/snowflake/snowpark/_internal/telemetry.py

```diff
@@ -428,6 +433,42 @@ def send_large_query_breakdown_telemetry(
         }
         self.send(message)
 
+    def send_post_compilation_stage_telemetry(
```

Comment:
maybe let's call send_query_compilation_summary_telemetry and add a comment about what is this telemetry sending